src/index.js

```
@@ -13,18 +13,20 @@ class Frecency {
   _idAttribute: string | Function;
 
   _localStorageEnabled: boolean;
+  _storageProvider: StorageProvider;
   _frecency: FrecencyData;
 
-  constructor({ key, timestampsLimit, recentSelectionsLimit, idAttribute }: FrecencyOptions) {
+  constructor({ key, timestampsLimit, recentSelectionsLimit, idAttribute, storageProvider }: FrecencyOptions) {
     if (!key) throw new Error('key is required.');
 
     this._key = key;
     this._timestampsLimit = timestampsLimit || 10;
     this._recentSelectionsLimit = recentSelectionsLimit || 100;
     this._idAttribute = idAttribute || '_id';
+    this._storageProvider = storageProvider || localStorage;
```

I foresee the errors being a little confusing if a user tries to use this in Node and doesn't provide a storage provider. Can we throw a specific error/assertion if this is Node and `storageProvider` is missing?

---

Added